mesonbuild/dependencies/python.py

```diff
@@ -271,6 +271,27 @@ def get_windows_link_args(self) -> T.Optional[T.List[str]]:
                         libpath = Path(f'python{vernum}.dll')
                 else:
                     libpath = Path('libs') / f'python{vernum}.lib'
+                    # For a debug build, pyconfig.h forces linking with
+                    # pythonX_d.lib (see meson#10776). This cannot be avoided
+                    # and won't work unless we also have a debug build of
+                    # Python itself. So error out here.
```

Comment:
We only warn, now, because pybind11.

Reply:
Good point, will update. What's the protocol around CI for tweaking a comment like this? Should I add `[skip actions]` or some such thing in the last commit, or re-run all CI jobs?

Reply:
Thanks. Comment updated now.

Reply:
Well really I'd like the changes squashed but then I can honestly just cancel CI and merge it directly. :) Skipping it via a commit message comment is fine, but secondary to the history IMO.

Reply:
okay, I'll do that now.